modules/auth/assembly-enabling-authentication.adoc

```diff
@@ -0,0 +1,35 @@
+[id='enabling-authentication']
```

Comment:
@hmanwani-rh does the id need to be `assembly-enabling-authentication` or is it ok as is? We might want to convert this to a title one day.

Reply:
I have looked at some other assemblies in the repository, and they do not have the prefix.